service/gradle/generators.gradle

```diff
@@ -0,0 +1,94 @@
+dependencies {
+    implementation 'io.swagger.core.v3:swagger-annotations'
+    runtimeOnly 'org.webjars.npm:swagger-ui-dist:4.9.0'
+    swaggerCodegen 'io.swagger.codegen.v3:swagger-codegen-cli'
+
+    // Jersey rest API client used by open api
+    // Version controlled by dependency management plugin
+    implementation 'org.glassfish.jersey.core:jersey-client'
+    implementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310'
+    implementation 'org.glassfish.jersey.media:jersey-media-json-jackson'
+    implementation 'org.glassfish.jersey.media:jersey-media-multipart'
+
+
+    // Versioned by Spring
+    implementation 'javax.validation:validation-api'
+    implementation 'org.webjars:webjars-locator-core'
+    annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
+}
+
+def artifactGroup = "${group}"
+
+// download calhoun openapi yaml
+def calhounSwaggerUrl = 'http://raw.githubusercontent.com/DataBiosphere/calhoun/c189957aa4d44e7573ec4d6ed800149484f7ebb3/static/api-docs.yaml'
```

Comment:
Is there an expression to get you the latest file? This looks like it is tied to a specific hash.
Maybe consider the github API?

Reply:
We currently do the same in axon UI https://github.com/verily-src/terra-ui/blob/master/package.json#L64. Perhaps the intention was to ensure that the url will always be valid if the repo gets updated.

Reply:
any reason you don't want to point to master branch?

https://raw.githubusercontent.com/DataBiosphere/calhoun/master/static/api-docs.yaml

Reply:
We point to a specific hash to ensure that we generate the client expected by axon server so that actions that call build won't fail (update submodule, creating and publishing release container).

Calhoun maintainer offered to review a change for calhoun to build a client and upload to artifactory. I can revert this work once that change is in. 👍🏼